src/web/components/Nav/ExpandedMenu/ExpandedMenuToggle/NoJSButton.tsx

```diff
@@ -0,0 +1,127 @@
+    right: 5px;
+    bottom: 48px;
+    ${from.mobileMedium} {
+        bottom: ${display === 'immersive' ? '3px' : '-3px'};
+        right: 5px;
+    }
+    ${from.mobileLandscape} {
+        right: 18px;
+    }
+    ${from.tablet} {
+        bottom: 3px;
+    }
+    ${from.desktop} {
+        display: none;
+    }
+`;
+
+const openExpandedMenu = (display: Display) => css`
+    ${headline.xsmall()};
+    font-weight: 300;
+    color: ${brandText.primary};
+    cursor: pointer;
+    display: none;
+    position: relative;
+    overflow: hidden;
+    border: 0;
+    background-color: transparent;
+    height: 48px;
+    padding-left: 9px;
+    padding-right: 20px;
+    ${from.desktop} {
+        display: block;
+        padding-top: ${display === 'immersive' ? '9px' : '5px'};
+        height: 42px;
+    }
+    :hover,
+    :focus {
+        color: ${brandAlt[400]};
+    }
+`;
+
+export const NoJSButton = ({
+    menuCheckboxId,
+    display,
+    isVeggieBurger,
+    children,
+}: Props) => {
+    // Supporting NoJS and accessibility is hard.
+
+    // We have therefore have added JS to help us make the page accessibile.
+    // Adding JS to onKeyDown as a fall back to replace onClick input checkbox
+    // helps us keep some accessiblity for keyboard only users.
+    // This is not a perfect solution as not all screen readers support JS
+    // https://webaim.org/projects/screenreadersurvey8/#javascript
+
+    return (
+        // We are using label and `htmlFor` prop to be able to toggle an input checkbox
+        // However this means that we are using a label as a button and lose out on
+        // browser accessiblity.
+
+        // We need Typescript to ignore the abnormal props we have added to the label
+        // But in JSX this can be sometimes a little difficult
+        // https://github.com/microsoft/TypeScript/issues/27552#issuecomment-427928685
+
+        /* eslint-disable @typescript-eslint/ban-ts-ignore, jsx-a11y/label-has-associated-control, @typescript-eslint/no-unused-expressions, react/no-unknown-property, jsx-a11y/no-noninteractive-element-to-interactive-role */
+        // @ts-ignore
+
+        <label
+            className={
+                isVeggieBurger
+                    ? veggieBurgerStyles(display)
+                    : openExpandedMenu(display)
+            }
+            aria-label="Toggle main menu"
+            key="OpenExpandedMenuButton"
+            htmlFor={menuCheckboxId}
+            data-link-name="nav2 : veggie-burger"
+            onKeyDown={e => {
+                // keyCode: 13 => Enter key
+                // keyCode: 32 => Space key
+                if (e.keyCode === 13 || e.keyCode === 32) {
+                    // @ts-ignore
+                    document && document.getElementById(menuCheckboxId).click();
+                }
+            }}
+            // @ts-ignore
+            tabindex={0}
+            role="button"
+            data-cy={isVeggieBurger ? 'veggie-burger' : 'show-more-button'}
```

Comment:
This couples the application javascript with the test but this is an anti-pattern that can lead to brittle tests.

https://docs.cypress.io/guides/references/best-practices.html#Selecting-Elements